spec/lib/rx/api/prescriptions_spec.rb

```diff
@@ -9,7 +9,7 @@
   describe 'prescriptions' do
     let(:post_refill_error) { File.read('spec/support/fixtures/post_refill_error.json') }
 
-    before(:each) do
+    before(:all) do
```

Comment:
Without this change, I was getting failures due to the obfuscation of session tokens when trying to regenerate the cassette - it was passing the obfuscated token value to MHV.